package.json

```diff
@@ -39,10 +39,12 @@
     "react-scripts": "^3.4.1"
   },
   "dependencies": {
+    "d3": "^5.16.0",
     "lodash": "^4.17.15",
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
-    "react-dom": "^16.13.1"
+    "react-dom": "^16.13.1",
+    "react-spring": "^8.0.27"
```

Comment:
what is this for?

Reply:
I think it must have been included when I was playing around with dependencies for d3. Ill remove this